packages/docusaurus-plugin-content-docs/src/versions/__tests__/index.test.ts

```diff
@@ -758,4 +758,136 @@ describe('readVersionsMetadata', () => {
       );
     });
   });
+
+  describe('versioned site with custom versions dir, pluginId=community', () => {
```

Comment:
It feels overkill to add all these new tests just for testing that particular case: lots of boilerplate for little being tested

Why not simply creating a new entry

```js
    it('works with relative versionedDocsPath', async () => {
      const {defaultOptions, defaultContext, vCurrent} = await loadSite();

      const versionsMetadata = await readVersionsMetadata({
        options: {...defaultOptions, versionedDocsPath: './community-versions'},
        context: defaultContext,
      });

      expect(versionsMetadata).toEqual([
        {
          ...vCurrent,
          contentPath: "...",
          sidebarFilePath: "...",
        },
      ]);
   )}
```

This should test it without so much ceremony

Also please test with absolute paths?